### PR TITLE
Cisco devices

### PR DESCRIPTION
**Pull request by francesco-noc**

### CODE REVIEW

**Comment by francesco-noc** [comment]
Sorry I clicked by mistake. 
I'm still having problems with the pickle files.

**Comment by danner26** [comment]
Hey, you can try grabbing the latest pickle files and adding those, that might solve the issue. If not starting a new branch with the changes will definitely fix it.

I am actively merging things so there is a chance the first options does not work

File: device-types/Cisco/C9500-24Y4C-A.yaml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    type: rj-45
+  - name: usb
+    type: usb-mini-b
+power-ports:
+  - name: PS-0
+    type: iec-60320-c14
+    maximum_draw: 650
+  - name: PS-1
+    type: iec-60320-c14
+    maximum_draw: 650
```
Comment (DanSheps) [inline device-types/Cisco/C9500-24Y4C-A.yaml:20]: Power ports should be modules for this model

File: device-types/Cisco/C9120AXI-E.yaml [hunk 2]
```diff
@@ -4,13 +4,15 @@ model: Catalyst 9120AXI-E
 slug: cisco-c9120axi-e
 part_number: C9120AXI-E
 u_height: 0
+
```
Comment (DanSheps) [inline device-types/Cisco/C9120AXI-E.yaml:7]: I don't think we need a line break here

File: device-types/Cisco/C9120AXI-E.yaml [hunk 3]
```diff
@@ -4,13 +4,15 @@ model: Catalyst 9120AXI-E
...
 is_full_depth: false
 interfaces:
   - name: WiFi
     type: ieee802.11ax
   - name: GigabitEthernet0
     type: 2.5gbase-t
     poe_mode: pd
+
```
Comment (DanSheps) [inline device-types/Cisco/C9120AXI-E.yaml:15]: I don't think we need a line break here

File: device-types/Cisco/IE-2000-4TS-G-L.yaml
Comment (DanSheps) [inline device-types/Cisco/IE-2000-4TS-G-L.yaml:1]: Please add weight

File: device-types/Cisco/IE-2000-16PTC-G-L.yaml
Comment (DanSheps) [inline device-types/Cisco/IE-2000-16PTC-G-L.yaml:1]: Please add weight

File: device-types/Cisco/IE-3000-4TC.yaml
Comment (DanSheps) [inline device-types/Cisco/IE-3000-4TC.yaml:1]: Please add weight

File: device-types/Cisco/N7K-C7009.yaml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+part_number: N7K-C7009
+u_height: 14
+is_full_depth: true
+subdevice_role: parent
+comments: '[Cisco Nexus 7000 Series Switches Data Sheet](https://www.cisco.com/c/en/us/products/collateral/switches/nexus-7000-series-switches/Data_Sheet_C78-437762.html)'
+console-ports:
+  - name: con0
+    type: rj-45
+  - name: con1
+    type: rj-45
```
Comment (DanSheps) [inline device-types/Cisco/N7K-C7009.yaml:14]: This should be part of the supervisor module

File: device-types/Cisco/N7K-C7009.yaml
Comment (DanSheps) [inline device-types/Cisco/N7K-C7009.yaml:1]: Please add weight

File: device-types/Cisco/WS-C3560-24PS-S.yaml
Comment (DanSheps) [inline device-types/Cisco/WS-C3560-24PS-S.yaml:1]: Please add weight

File: device-types/Cisco/WS-C3560G-24PS-S.yaml
Comment (DanSheps) [inline device-types/Cisco/WS-C3560G-24PS-S.yaml:1]: Please add weight

File: device-types/Cisco/WS-C3560G-24TS-S.yaml
Comment (DanSheps) [inline device-types/Cisco/WS-C3560G-24TS-S.yaml:1]: Please add weight

File: device-types/Cisco/WS-C3750X-24S-E.yaml
Comment (DanSheps) [inline device-types/Cisco/WS-C3750X-24S-E.yaml:1]: Please add weight

File: device-types/Cisco/WS-C3850-24S-E.yaml
Comment (DanSheps) [inline device-types/Cisco/WS-C3850-24S-E.yaml:1]: Please add weight

**Review by DanSheps — CHANGES_REQUESTED** [review]

**Comment by francesco-noc** [comment]
changes made!

**Review by DanSheps — APPROVED** [review]

**Review by DanSheps — CHANGES_REQUESTED** [review]
Please fix merge conflicts and other issues

**Comment by danner26** [comment]
Hey @francesco-noc can you please resolve this PR issues or close the PR? Thank you

**Comment by danner26** [comment]
Uh something definitely got messed up here. You ended up making changes to a lot of files I am guessing you didnt mean to. Not sure how it happened

**Comment by francesco-noc** [comment]
🤦‍♂️

**Pull request closed without merging**